code/modules/admin/admin_verbs.dm

```diff
@@ -296,6 +296,7 @@ GLOBAL_PROTECT(admin_verbs_hideable)
 				add_verb(/client/proc/play_web_sound)
 		if(rights & R_SPAWN)
 			add_verb(GLOB.admin_verbs_spawn)
+		reset_badges()
```

Comment:
Should this really be in this proc?

Reply:
Yes, it resets the badge cache allowing the badges to be reloaded